resources/nethealth/nethealth.yaml

```diff
@@ -161,17 +157,10 @@ spec:
               add:
               - NET_RAW
           imagePullPolicy: Always
-          volumeMounts:
```

Comment:
I would double-check with @knisbet (if you haven't yet) whether it's safe to remove this.

Reply:
We've already discussed it in slack with @bernardjkim and @knisbet.